setup.cfg

```diff
@@ -16,8 +16,8 @@ exclude =
 max-complexity = 10
 max-line-length = 88
 select = C,E,F,W,B,B950
-# C812, C815, W503 clash with black, F723 false positive
-ignore = E501,C812,C815,C816,W503,F723
+# C812, C815, E203, W503 clash with black, F723 false positive
+ignore = E203,E501,C812,C815,C816,W503,F723
```

Comment:
Added this because of line 251 in codegen.py

Reply:
why? can u remove whitespace instead?

Reply:
black add space [here](https://github.com/aws-cloudformation/cloudformation-cli-python-plugin/pull/166/files#diff-65ed84246aa8d407e713019bad551971275dd9cf3ce9ccf60b2bb5a815e05a81R251) and flake8 complains.